teams/wg-rustc-dev-guide.toml

```diff
@@ -29,8 +29,8 @@ alumni = [
 orgs = ["rust-lang"]
 
 [website]
-name = "Rustc Dev Guide working area"
-description = "Making the compiler easier to learn by maintaining and improving the Rustc Dev Guide"
+name = "rustc-dev-guide working area"
```

Comment:
The title I see on [that website](https://rustc-dev-guide.rust-lang.org/) is "Rust Compiler Development Guide". Perhaps did you mean to use that? `rustc-dev-guide` is (iiuc) the domain name so imo not very clear.

Reply:
I went for the shorter name for the title to avoid it being too wordy, and added the formal name on the description